src/rust/plugin-registry/src/static_files/hax_docker_analyzer.nomad

```diff
@@ -0,0 +1,284 @@
+  type        = number
+  default     = 1
+  description = "The number of instances of the plugin to run."
+}
+
+variable "aws_account_id" {
+  type        = string
+  description = "The account ID of the aws account that holds onto the plugin binaries."
+}
+
+variable "plugin_runtime_image" {
+  type        = string
+  description = "The container that will load and run the plugin"
+}
+
+variable "plugin_execution_sidecar_image" {
+  type        = string
+  description = "The container that will load and run the Generator Executor or Analyzer Executor"
+}
+
+variable "graph_query_proxy_image" {
+  description = "Container image for running Graph Query Proxy. Only specify this for Analyzers."
+  type        = string
+  default     = null
+}
+
+variable "rust_log" {
+  type        = string
+  description = "Controls the logging behavior of Rust-based services."
+}
+
+variable "observability_env_vars" {
+  type        = string
+  description = <<EOF
+With local-grapl, we have to inject env vars for Opentelemetry.
+In prod, this is currently disabled.
+EOF
+}
+
+job "grapl-plugin" {
+  datacenters = ["dc1"]
+  namespace   = "plugin-${var.plugin_id}"
+  type        = "service"
+
+  reschedule {
+    # Make this a one-shot job
+    attempts = 0
+  }
+
+  # We'll want to make sure we have the opposite constraint on other services
+  # This is set in the Nomad agent's `client` stanza:
+  # https://www.nomadproject.io/docs/configuration/client#meta
+  constraint {
+    attribute = "${meta.is_grapl_plugin_host}"
+    value     = true
+  }
+
+  group "plugin-execution-sidecar" {
+    count = var.plugin_count
+
+    network {
+      mode = "bridge"
+      // TODO i think? possibly cargo culted?
+      // dns {
+      //   servers = local.dns_servers
+      // }
+      port "plugin-execution-sidecar" {}
+      port "graph-query-proxy" {}
+    }
+
+    service {
+      name = "plugin-execution-sidecar-${var.plugin_id}"
+      tags = [
+        "plugin-execution-sidecar",
+        "tenant-${var.tenant_id}",
+        "plugin-${var.plugin_id}"
+      ]
+
+      connect {
+        sidecar_service {
+          proxy {
+            upstreams {
+              destination_name = "plugin-work-queue"
+              # port unique but arbitrary - https://github.com/hashicorp/nomad/issues/7135
+              local_bind_port = 1000
+            }
+
+            upstreams {
+              destination_name = "plugin-${var.plugin_id}"
+              # port unique but arbitrary - https://github.com/hashicorp/nomad/issues/7135
+              local_bind_port = 1001
+            }
+
+            upstreams {
+              destination_name = "graph-query"
+              # port unique but arbitrary - https://github.com/hashicorp/nomad/issues/7135
+              local_bind_port = 1002
+            }
+          }
+        }
+      }
+    }
+
+    # The execution task pulls messages from the plugin-work-queue and
+    # sends them to the plugin
+    task "plugin-execution-sidecar" {
```

Comment:
maybe "analyzer-execution-sidecar"? If so, we should name the one for generators in a corresponding fashion.

Reply:
sure